django/db/backends/utils.py

```diff
@@ -114,6 +114,27 @@ def _executemany(self, sql, param_list, *ignored_wrapper_args):
             return self.cursor.executemany(sql, param_list)
 
 
+def _log_query(connection, start, stop, sql, params=None):
```

Comment:
This helper removes duplication for query logging. Could be a commit on its own, assuming the approach is considered decent.